pom.xml

```diff
@@ -100,7 +111,7 @@
         <glassfish.json.version>1.1.6</glassfish.json.version>
         <wildfly.common.version>1.5.2.Final</wildfly.common.version>
         <ua-parser.version>1.4.3</ua-parser.version>
-        <picketbox.version>5.0.3.Final</picketbox.version>
+        <picketbox.version>5.0.3.Final-redhat-00005</picketbox.version>
```

Comment:
@iankko Why we need this redhat version?

Reply:
@martin-kanis It is exactly the version, which is bundled inside the wildfly-20.0.0.Final in the directory modules/system/layers/base/org/picketbox/main . Hence I suppose we need this version?

Reply:
I see. It makes sense now.

Reply:
@martin-kanis Yes. it's the version that comes from Wildfly 20 (that's why it got updated).